modules/monitoring-reviewing-monitoring-dashboards-developer.adoc

```diff
@@ -6,16 +6,20 @@
 [id="reviewing-monitoring-dashboards-developer_{context}"]
 = Reviewing monitoring dashboards as a developer
 
-In the *Developer* perspective, you can view dashboards relating to a selected project. You must have access to monitor a project to view dashboard information for it.
+In the *Developer* perspective, you can view dashboards relating to a selected project.
+
+[NOTE]
+====
+In the *Developer* perspective, you can view dashboards for only one project at a time.
+====
 
 .Prerequisites
 
-* You have access to the cluster as a developer or as a user.
```

Comment:
@juzhao I removed this prereq as it seems redundant. We do not mention in any other section being a dev as a prerequisite. Also, being "a user" seemed very vague. Let me know in case you think it should stay. Thanks!

Reply:
@eromanova97 I think keep it is fine, any user who create the project can view the project dashboard